### PR TITLE
Minor readme updates

### PR DESCRIPTION
**Pull request by oak9-ashah**
Minor README cleanup

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -167,3 +147,31 @@ To view your blueprint's validation results side-by-side with the oak9 blueprint
...
+
+It is most likely there is a dependency conflict with the Tython packages you recently installed. By default, pip does not flag any dependency conflicts and will download multiple versions of a dependency into your project, resulting in errors. One approach to dealing with this (our recommended approach) is using an enhanced dependency management tool such as a virtual enviroments. Setting this up on your personal machine only takes a few minutes and will likely save you hours of troubleshooting pip versioning issues in the future, just follow the steps below:
+
+1] Upgrade pip
+`pip install -i https://pypi.org/simple --upgrade pip`
+
+2] Install virtualenv
+`pip3 install -i https://pypi.org/simple virtualenv`
+
+3] Get the path for you Python 3.11 installation
```
Comment (bnicoll-oak9) [inline README.md:167]:
```suggestion
3] Get the path for your Python 3.11 installation
```

File: README.md [hunk 2]
```diff
@@ -167,3 +147,31 @@ To view your blueprint's validation results side-by-side with the oak9 blueprint
...
+`which -a python3`
+
+4] Create a virtual environment with this version called **tython-venv**
+`virtualenv -p /Library/Frameworks/Python.framework/Versions/3.11/bin/python3 tython-venv`
+
+5] Activate your new enviroment
```
Comment (bnicoll-oak9) [inline README.md:173]:
```suggestion
5] Activate your new environment
```

File: README.md [hunk 3]
```diff
@@ -167,3 +147,31 @@ To view your blueprint's validation results side-by-side with the oak9 blueprint
...
+
+## Help setting up your Python virtual environment
+
+If you've done the pip install and still find yourself facing one or both of the following errors -
+
+`Issue finding python runtime and/or version`
+
+`[Error] oak9 Tython framework package is not installed. Be sure to run "pip install -r requirements.txt`
+
+It is most likely there is a dependency conflict with the Tython packages you recently installed. By default, pip does not flag any dependency conflicts and will download multiple versions of a dependency into your project, resulting in errors. One approach to dealing with this (our recommended approach) is using an enhanced dependency management tool such as a virtual enviroments. Setting this up on your personal machine only takes a few minutes and will likely save you hours of troubleshooting pip versioning issues in the future, just follow the steps below:
```
Comment (bnicoll-oak9) [inline README.md:159]:
```suggestion
It is most likely there is a dependency conflict with the Tython packages you recently installed. By default, pip does not flag any dependency conflicts and will download multiple versions of a dependency into your project, resulting in errors. One approach to dealing with this (our recommended approach) is using an enhanced dependency management tool such as a virtual environments. Setting this up on your personal machine only takes a few minutes and will likely save you hours of troubleshooting pip versioning issues in the future, just follow the steps below:
```

Comment (morgan-oak9) [reply README.md:159]: this word is my worst enemy

Comment (morgan-oak9) [reply README.md:159]: emeny

**Review by morgan-oak9 — APPROVED** [review]

**Pull request merged**